CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Changelog for MassiveBuildBundle
 ================================
 
+* [0.5.6 (2023-04-05)](https://github.com/massiveart/MassiveBuildBundle/releases/tag/0.6.0)
```

Comment:
0.5.6 or 0.6.0 ? in the url you are using 0.6.0

Reply:
fixed should be 0.5.6